pkg/minikube/driver/driver.go

```diff
@@ -214,6 +217,9 @@ func Suggest(options []registry.DriverState) (registry.DriverState, []registry.D
 // Status returns the status of a driver
 func Status(name string) registry.DriverState {
 	d := registry.Driver(name)
+	if d.Empty() {
+		exit.WithCodeT(exit.Unavailable, "The driver '{{.driver}}' is not supported on {{.os}}", out.V{"driver": name, "os": runtime.GOOS})
```

Comment:
I don't love having libraries panic - is it possible to move this logic closer to the `main` package?

Alternatively, could this return a driver state that represents the issue and exits sooner in main?

Reply:
moving it to main was easy enough, done